infrastructure/deploy.sh

```diff
@@ -269,6 +269,18 @@ mkdir -p /tmp/opencrvs/infrastructure/hearth-plugins
 (cd /tmp/opencrvs/infrastructure/hearth-plugins && curl -O https://raw.githubusercontent.com/opencrvs/opencrvs-core/$VERSION/hearth-plugins/checkDuplicateTask.js)
 cd /tmp/
 
+# Copy authorized keys
+cp $BASEDIR/.ssh/authorized_keys /tmp/opencrvs/infrastructure/authorized_keys
+
+rotate_authorized_keys() {
+  # file exists and has a size of more than 0 bytes
+  if [ -s "/tmp/opencrvs/infrastructure/authorized_keys" ]; then
+    ssh $SSH_USER@$SSH_HOST 'cat /tmp/opencrvs/infrastructure/authorized_keys > ~/.ssh/authorized_keys'
+  else
+    echo "File /tmp/opencrvs/infrastructure/authorized_keys is empty. Did not rotate authorized keys!"
+  fi
+}
+
```

Comment:
not sure if this can be called rotating as we are just copying the contents of the file? :sweat_smile:

Reply:
well we are overriding it every time with the latest values. Don't know if rotating is a good word. Maybe just override_authorized_keys